resume.md

```diff
@@ -1,49 +1,47 @@
-# FIRST LAST NAME
+# Daniel Qian
 
-(123) 456 - 7890 | email@gmail.com
+(925) 922 - 4281 | danielq3@illinois.edu
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+github.com/DAQIANN | linkedin.com/danielq5/
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
-
-## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
-* *...*
+**University of Illinois Urbana Champaign** *May 2023* *GPA 3.89* *Dean's List*
+* Bachelor of Science in Mathematics and Computer Science 
+* Minor in Statistics 
+* **Coursework**: Data Structures, Numerical Methods I, Software Design Studio, Discrete Structures, Introduction to Computer Science, Linear Algebra, Statistical Analysis, Statistics and Probability I
```

Comment:
Would use the generic name for these classes / would remove introduction classes
* Numerical Methods I (not sure what this is)
* Software Design Studio (not sure what this is)
* ~~Introduction to Computer Science~~